cmd/spire-agent/cli/run/run.go

```diff
@@ -97,7 +97,9 @@ type experimentalConfig struct {
 	SyncInterval  string `hcl:"sync_interval"`
 	TCPSocketPort int    `hcl:"tcp_socket_port"`
 
-	UnusedKeys []string `hcl:",unusedKeys"`
+	UnusedKeys            []string `hcl:",unusedKeys"`
+	MaxSvidCacheSize      int      `hcl:"max_svid_cache_size"`
+	SVIDCacheExpiryPeriod string   `hcl:"svid_cache_expiry_interval"`
```

Comment:
HCL name and internal field should match

Reply:
After reviewing how expiry is used, I'm not convinced it's really needed. I have a couple concerns with attaching expiries to SVIDs in the cache:

1. In the end, it's adding another indirect way to influence the maximum cache size. I feel that adding expiry to cache entries creates some unneeded complexity without an obvious value-add from just tweaking the max cache size configurable.
2. It's not clear how a user would determine what to set this to. The cache expiry also has an indirect connection with SVID TTLs, which are configurable at both the Server and entry level, which further complicates what this value should be set to. This feels like a potential footgun that may cause unintended consequences such as cache churn or cache bloat when combined with the max SVID cache configuration and SVID TTLs.

Reply:
I think the initial intention was "what if SVID _foo_ loses all subscribers, but a new subscriber comes up soon after", and trying to avoid an extra signing operation. However since _foo_ was recently subscribed to this would only be a concern if all other SVIDs up to (or past) max cache were also being subscribed to.
In that heavy subscriber load scenario, one extra SVID signing probably won't be a concern.

Reply:
Removed `SVIDCacheExpiryPeriod` from config and implementation.